pages/reference/base-images/base-images.md

```diff
@@ -108,92 +107,97 @@ CMD ["node", "main.js"]
+Currently, balenalib supports the following OS distributions and Language stacks, if you would like to see others added, create an issue on the [balena base images repo]({{ $links.githubLibrary }}/base-images/issues).
+
+| Distribution | Default                      | Supported Architectures                      |
+| ------------ | ---------------------------- | -------------------------------------------- |
+| Debian       | Debian GNU/Linux 10 (buster) | armv5e, armv6, armv7hf, aarch64, amd64, i386 |
+| Alpine       | Alpine Linux v3.10           | armv6, armv7hf, aarch64, amd64, i386         |
+| Ubuntu       | 18.04 LTS (bionic)           | armv7hf, aarch64, amd64, i386                |
+| Fedora       | Fedora 30                    | armv7hf, aarch64, amd64, i386                |
+
+| Language | Default | Supported Architectures                      |
+| -------- | ------- | -------------------------------------------- |
+| Node.js  | v13.5.0 | armv6, armv7hf, aarch64, amd64, i386         |
+| Python   | v2.7.17 | armv5e, armv6, armv7hf, aarch64, amd64, i386 |
+| OpenJDK  | v11-jdk | armv7hf, aarch64, amd64, i386, armv6         |
+| Go       | 1.13.4  | armv7hf, aarch64, amd64, i386, armv6         |
+| Dotnet   | 3.1-sdk | armv7hf, aarch64, amd64                      |
 
-| Distribution | Default                | Supported Architectures                      |
-|---------|------------------------------|----------------------------------------------|
-| Debian  | Debian GNU/Linux 9 (stretch) | armv5e, armv6, armv7hf, aarch64, amd64, i386 |
-| Alpine  | Alpine Linux v3.10            | armv6, armv7hf, aarch64, amd64, i386 		|
-| Ubuntu  | 18.04 LTS (Bionic Beaver)    | armv7hf, aarch64, amd64, i386                |
-| Fedora  | Fedora 29 (Twenty Nine)      | armv7hf, aarch64, amd64, i386                |
+#### Notes
 
-| Language | Default  	                  | Supported Architectures                      |
-|---------|------------------------------|----------------------------------------------|
-| Node.js | v12.7.0                      | armv6, armv7hf, aarch64, amd64, i386         |
-| Python  | v2.7.16                       | armv5e, armv6, armv7hf, aarch64, amd64, i386 |
-| OpenJDK | v1.8.0_181                    | armv7hf, aarch64, amd64, i386, armv6         |
-| Go      | 1.12.7                        | armv7hf, aarch64, amd64, i386, armv6         |
-| Dotnet  | 2.2-sdk                       | armv7hf, aarch64, amd64                |
+Devices with a [device type](/reference/base-images/devicetypes/) of `raspberry-pi` (Raspberry Pi1 and Zero) will be built from `balenalib/rpi-raspbian` and will be [Raspbian]({{ $links.githubLibrary }}/resin-rpi-raspbian) base images. Raspberry Pi2 and Pi3 base images have the Raspbian package source added and Raspbian userland pre-installed.
```

Comment:
"Raspberry Pi2 and Pi3 base images have the Raspbian package source added and Raspbian userland pre-installed."

Is this true for just Debian on Raspberry Pi, or for the other OSes too? (I guess it wouldn't make sense to add the Raspbian package source to Fedora?)

Also, I think "Raspberry Pi2 and Pi3" ought to be "Raspberry Pi2 and Raspberry Pi3" ? :man_shrugging:

Reply:
...and there's also the Raspberry Pi 4 of course :wink: Maybe "Raspberry Pi2 and above" ?

Reply:
No for Pi4, as we only offer 64-bit support (and I need to explicitly mention that for Pi3 variant).  I'm checking the distros and looks like only Debian adds Raspbian package source via https://github.com/balena-io-library/base-images/blob/master/balena-base-images/device-base/raspberrypi3/debian/jessie/build/Dockerfile. Maybe @nghiant2710 can confirm this though before I update.

Reply:
This issue is also related as this doesn't work for `debian:sid` variant https://github.com/balena-io-library/base-images/issues/602

Reply:
Interesting - why no Raspbian on RPi4 on balena, when the images at https://www.raspberrypi.org/downloads/raspbian/ support the RPi4?

Reply:
There is no balena `raspberrypi4` device unlike the Pi3, there is only `raspberrypi4-64`.